index.html

```diff
@@ -41,19 +40,158 @@ <h1 id="idh1">
+            <h1>Multi-Post <br> Stories</h1>
+            <div class="info">
+                <h4>CANOPY</h4>
+                <ul class="extra">
+                    <li class="li1"><span>Back End Dev</span></li>
+                    <li class="li2"><span>2015</span></li>
+                </ul>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="skills">
+                <li>
+                    <a href="#"><button class="b1">html</button></a>
+                </li>
+                <li>
+                    <a href="#"><button class="b2">css</button></a>
+                </li>
+                <li>
+                    <a href="#"><button class="b3">javascript</button></a>
+                </li>
+            </ul>
+            <a href="#"><button class="b4">See Project</button></a>
+        </section>
+        <section class="work s3">
+            <img src="./images/s3.svg" alt="tonic">
+            <h1>Tonic</h1>
+            <div class="info">
+                <h4>CANOPY</h4>
+                <ul class="extra">
+                    <li class="li1"><span>Back End Dev</span></li>
+                    <li class="li2"><span>2015</span></li>
+                </ul>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="skills">
+                <li>
+                    <a href="#"><button class="b1">html</button></a>
+                </li>
+                <li>
+                    <a href="#"><button class="b2">css</button></a>
+                </li>
+                <li>
+                    <a href="#"><button class="b3">javascript</button></a>
+                </li>
+            </ul>
+            <a href="#"><button class="b4">See Project</button></a>
+        </section>
+        <section class="work s4">
+            <img src="./images/s4.svg" alt="tonic">
+            <h1>Multi-Post <br> Stories</h1>
+            <div class="info">
+                <h4>CANOPY</h4>
+                <ul class="extra">
+                    <li class="li1"><span>Back End Dev</span></li>
+                    <li class="li2"><span>2015</span></li>
+                </ul>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="skills">
+                <li>
+                    <a href="#"><button class="b1">html</button></a>
+                </li>
+                <li>
+                    <a href="#"><button class="b2">css</button></a>
+                </li>
+                <li>
+                    <a href="#"><button class="b3">javascript</button></a>
+                </li>
+            </ul>
+            <a href="#"><button class="b4">See Project</button></a>
+        </section>
+        <section class="about-myself">
+            <h1>About <br> Myself</h1>
```

Comment:
- [ ] h2 tag here as well

Reply:
Done @tufoinnkuo10 . I've already changed it to an h2 tag.